jetty-util/src/main/java/org/eclipse/jetty/util/ArrayTernaryTrie.java

```diff
@@ -190,7 +201,7 @@ public boolean put(String s, V v)
                 if (t == _rows)
                 {
                     _rows++;
-                    if (_rows >= _key.length)
+                    if (_rows > _key.length)
```

Comment:
This change will make line 209 throw `ArrayIndexOutOfBoundsException` if `_rows == _key.length` which happens when `t == Character.MAX_VALUE`.

Reply:
I will fix it~